src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -725,6 +728,36 @@ private NodeSearchElementViewModel MakeNodeSearchElementVM(NodeSearchElement ent
             return elementVM;
         }
 
+        private NodeCategoryViewModel GetCategoryViewModel(IEnumerable<string> categories)
```

Comment:
We could make this `static` by passing in `libraryRoot`, it does sound like a helper method that we can potentially add unit tests for.